mindmeld/components/_config.py

```diff
@@ -42,11 +42,12 @@
 
 DEFAULT_DOMAIN_CLASSIFIER_CONFIG = {
     "model_type": "text",
-    "model_settings": {"classifier_type": "logreg",},
+    "model_settings": {"classifier_type": "logreg", },
     "param_selection": {
         "type": "k-fold",
         "k": 10,
-        "grid": {"fit_intercept": [True, False], "C": [10, 100, 1000, 10000, 100000]},
+        "grid": {"solver": ["liblinear"],
```

Comment:
I dont think it makes sense to add a single entry to the grid payload. Instead, we should just default load `liblinear ` into the `logreg` implementation.

Reply:
What is the reason to keep `liblinear` as our default? We have had our own issues with liblinear and did switch to lbfgs for some apps at that time. I don't remember now if that affected how the other params are specified, but lbfgs is a reasonable default.

Reply:
The old version uses liblinear as default but the new version uses lbfgs. If this is not a hard requirement, then I can remove any places where this is enforced. I initially added it cause I thought this was the solver giving the best result for the default use cases. Please do let me know whether it's needed @vijay120 @serapio

Reply:
after reviewing the sklearn documentation, I see that the choice of liblinear vs lbfgs does interact with the hyper-params `penalty`, `fit_intercept` and `multi_class`, e.g. `penalty=l1` is only valid with liblinear, and `penalty=none` is not valid with liblinear. So existing configs that were implicitly assuming liblinear might degrade performance switching to lbfgs, but retrained models with default params or with params appropriate to lbfgs should be fine (and existing trained models would not be affected). I'm inclined to just warn in the release notes how the change in default solver is relevant to the other hyper-parameters, to keep the configs simpler and consistent with sklearn defaults.

Reply:
Fixed as discussed. I've let the default go to lbfgs and changed the kwik_e_mart settings to be compatible with the tests.